openwisp-openwrt-monitoring/files/monitoring.init

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=15
+USE_PROCD=1
+PROG = "/usr/sbin/monitoring"
```

Comment:
/usr/sbin/monitoring > /usr/sbin/openwisp_monitoring

please get the name right, double check every line.

`openwisp_monitoring` for the agent, `netjson_monitoring` for the command which returns NetJSON output